test/sdsify.test.ts

```diff
@@ -443,11 +450,20 @@ describe("Functional tests for the sdsify function", () => {
             .on("end", () => {});
 
         // Execute function
-        sdsify(input, output, STREAM_ID, undefined, undefined, BAD_SHAPE_1);
         try {
+            await sdsify(
+                input,
+                output,
+                STREAM_ID,
+                undefined,
+                undefined,
+                BAD_SHAPE_1,
+            );
+
             // Push some data in
-            expect(await input.push(INPUT_1)).toThrow(Error);
+            expect(true).toBeFalsy();
```

Comment:
hmm?

Reply:
The call to sdsify should fail, thus adding ugly check that an exception was thrown :(